schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+	id INT NOT NULL,
```

Comment:
[OPTIONAL] good job by making the id not null I suggest also setting it as primary key for your table
PRIMARY KEY (id)